development/extensions/tutorial_testing.rst

```diff
@@ -879,8 +879,8 @@ first file is the Travis CI configuration file, ``.travis.yml``:
       - travis/prepare-phpbb.sh $PHPBB_BRANCH
       - cd ../../phpBB3
       - travis/prepare-extension.sh $EXTNAME $PHPBB_BRANCH
-      - travis/setup-phpbb.sh $DB $TRAVIS_PHP_VERSION
-      - sh -c "if [ '$EPV' == '1' -a '$NOTESTS' == '1' ]; then cd phpBB; composer remove sami/sami --update-with-dependencies --dev --no-interaction; composer require phpbb/epv:dev-master --dev --no-interaction --ignore-platform-reqs; cd ../; fi"
+      - travis/setup-phpbb.sh $DB $TRAVIS_PHP_VERSION $NOTESTS
+      - ../<authorname>/<reponame>/travis/prepare-epv.sh $EPV $NOTESTS
```

Comment:
Please undo this. Instead just move the EPV line to here, so the EPV script runs last. This section should look like the skeleton ext:

```
install:
  - travis/prepare-phpbb.sh $PHPBB_BRANCH
  - cd ../../phpBB3
  - travis/prepare-extension.sh $EXTNAME $PHPBB_BRANCH
  - travis/setup-phpbb.sh $DB $TRAVIS_PHP_VERSION $NOTESTS
  - sh -c "if [ '$EPV' = '1' -a '$NOTESTS' = '1' ]; then cd phpBB; composer remove sami/sami --dev --no-interaction; composer require phpbb/epv:dev-master --dev --no-interaction --ignore-platform-reqs; cd ../; fi"
```